src/Analysis/Codelyzer.Analysis/CSharpCodeAnalyzer.cs

```diff
@@ -39,6 +40,35 @@ public override async Task<List<AnalyzerResult>> AnalyzeSolution(string solution
             return await Analyze(solutionPath);
         }
 
+        ///<inheritdoc/>
+        public override CodeGraph GenerateGraph(List<AnalyzerResult> analyzerResults)
```

Comment:
If we were to do this for a different type of analyzer? Would the code be reused? Or is this specific to CSharpCodeAnalyzer

Reply:
Yes we can reuse because it's using the codelyzer results and not bound to any specific Roslyn objects